tests/integration/serverless/package.json

```diff
@@ -8,7 +8,7 @@
     "version": "serverless --version"
   },
   "devDependencies": {
-    "serverless": "^2.35.0",
+    "serverless": "2.48.0",
```

Comment:
I do not know what I did wrong when pulling from master. 
I'm used to run:
`git pull origin master --rebase` (I used to perform merge commits before)
`git push`
:tada: 

It is the first time the commits already merged in master show in the diff! Any advice/tip/hint it is appreaciated!

Reply:
I guess that should be fine when we squash-merge the PR - no worries.

What you have above should usually work. To be on the safe side, you could also try these steps manually:
```
git checkout master
git pull
git checkout your-branch
git rebase master
git push origin your-branch
```

---
Anyway. Thanks for the PR @GonzaloSaad ! 👍 - should be good to merge.